misc/git-hook/necessary/precommit/check/indent.bash

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+. $GITDIR/hooks/functions.bash
+
+changed_files=`getChangedFiles`
+issue_found=0
+
+IFS=$'\n' # bash specific
+for file in $changed_files
+do
+	result=`cat "$file" | grep -P -n '^[\t]*(    | {1,3}\t)'`
+
+	if [ -n "$result" ]
+	then
+		echo "file with indent issue: $file"
+		echo "$result"
```

Comment:
Might be wise to not output this as this can become quite noisy in log output.

Reply:
OK, but in common you want to know which lines has issues and must be fixed. If the codebase is without issues the noise should be low.

Reply:
Should we limit the output? 8 issues per file for example?